### PR TITLE
Fix & small improvement

### PR DESCRIPTION
**Pull request by victor-paumier**
- remove fixBdd
- fix delete user
- fix section on user & buddy list as admin
- improve sorting of matching lists
- add "contact my section" on dashboard
- mail report improvement

### CODE REVIEW

File: assets/js/matching.js [hunk 1]
```diff
@@ -16,7 +16,12 @@ $(document).ready(() => {
         const dataTable = $('#matching_internationals');
 
         if (dataTable !== undefined) {
-            tools.initDatatable(dataTable, { order: [[6, 'asc']] });
+            tools.initDatatable(dataTable, {
+                order: [[2, 'asc']],
+                columnDefs: [
+                    {type: "date-dd-MM-yyy", targets: [1, 2]}
```
Comment (JeremieSamson) [inline assets/js/matching.js]: maybe `date-dd-MM-yyy` as global var ?

File: assets/js/matching.js [hunk 2]
```diff
@@ -46,7 +49,12 @@ $(document).ready(() => {
         const menteeTable = $('#selected_international');
 
         if (dataTable !== undefined) {
-            tools.initDatatable(dataTable, { order: [[0, 'desc']] });
+            tools.initDatatable(dataTable, {
+                order: [[0, 'desc']],
+                columnDefs: [
+                    {type: "date-dd-MM-yyy", targets: [2, 3]}
```
Comment (JeremieSamson) [inline assets/js/matching.js]: same here

File: assets/sass/_base.scss [hunk 3]
```diff
@@ -183,3 +183,7 @@ hr.lightblue {
 .f-red {
   color: $red;
 }
+
+.f-black {
+  color: black;
```
Comment (JeremieSamson) [inline assets/sass/_base.scss]: don't you have a `$black` :thinking: ?

File: src/Model/BuddyListResponse.php [hunk 4]
```diff
@@ -55,7 +55,7 @@ public function configure(object $entity, array $options): array
             } else {
                 $confirmedStatus = $this->translator->trans('buddy.confirmed.none');
             }
-
+            $this->section = $mentor->getSection()->getName();
```
Comment (JeremieSamson) [inline src/Model/BuddyListResponse.php]: shouldn't this be name `sectionNamed` ?

File: src/Model/UserListResponse.php [hunk 5]
```diff
@@ -106,6 +106,7 @@ public function configure(object $entity, array $options): array
             $this->nbBuddy = $user->getNbBuddy();
             $this->archive = ($user->getArchive()) ? $this->translator->trans('yes') : $this->translator->trans('no');
             $this->roles = $roles;
+            $this->section = $user->getSection()->getName();
```
Comment (JeremieSamson) [inline src/Model/UserListResponse.php]: same

File: assets/sass/_base.scss (hunk 3 above)
Comment (victor-paumier) [reply assets/sass/_base.scss]: not but is $black: black very useful?

File: src/Model/BuddyListResponse.php (hunk 4 above)
Comment (victor-paumier) [reply src/Model/BuddyListResponse.php]: ok, but line 141 will be 'section' => $this->sectionNamed, good for u?

File: assets/js/matching.js [hunk 6]
```diff
@@ -2,6 +2,8 @@ import tools from './tools/tools';
 import Routing from './tools/Routing';
 
 $(document).ready(() => {
+    const dateDataTableType = "date-dd-MM-yyy";
```
Comment (JeremieSamson) [inline assets/js/matching.js]: can't you put it in a more high level ?

File: assets/sass/helpers/_colors.scss [hunk 7]
```diff
@@ -8,4 +8,6 @@ $light-grey: #f8f9fd;
 $dark-grey: #3C3C3B;
 $green: #00cf63;
 $yellow: #F9FF00;
-$red: #cf0000;
\ No newline at end of file
+$red: #cf0000;
+$black: black;
+$white: white;
```
Comment (JeremieSamson) [inline assets/sass/helpers/_colors.scss]: line break plz

File: assets/js/matching.js (hunk 6 above)
Comment (victor-paumier) [reply assets/js/matching.js]: before document ready??

**Pull request merged**